src/MongoDB.Driver.Core/Core/Servers/ServerDescription.cs

```diff
@@ -540,11 +576,13 @@ public ServerDescription WithHeartbeatException(Exception heartbeatException)
             return new ServerDescription(
                 _serverId,
                 _endPoint,
+                "HeartbeatFailed",
```

Comment:
Consider adding an explicit argument name before the value

Reply:
Sure... originally I had made `reasonChanged` a mandatory argument in which case the parameter name was less important.

Reply:
done